service/controller/clusterapi/v30/cloudconfig/tccp.go

```diff
@@ -57,7 +57,7 @@ func (t *TCCP) Render(ctx context.Context, cr cmav1alpha1.Cluster, clusterCerts
 		params.DisableEncryptionAtREST = true
 		// Ingress controller service remains in k8scloudconfig and will be
 		// removed in a later migration.
-		params.DisableIngressControllerService = false
+		params.DisableIngressControllerService = true
```

Comment:
We should also update the comment. Sorry I missed that first time.


> Ingress Controller service is not created via ignition. It is created by the Ingress Controller app if it is installed in the tenant cluster.

Reply:
Damn looks like it also affected the golden file test since this manifest is removed.

Reply:
yep, fixing tests, thanks for comment text too!